features/testdata/expecteddata/onlySICResponse.json

```diff
@@ -1,5 +1,5 @@
 {
-    "query": "01230",
+    "query": "",
```

Comment:
are these all supposed to be empty?

Reply:
Yes, since the new Implementation removes all special characters, all small words (in, on, a, etc.) and all SIC and OAC codes as they're not needed when presenting the query to Berlin and Category so from the tests which included OAC and SIC codes (but tested something else) I just removed the query

Reply:
Ah OK, makes sense.